src/app/functions/server/featureFlags.ts

```diff
@@ -34,5 +46,19 @@ export async function getFlag(name: string): Promise<FeatureFlag | undefined> {
     owner: data.owner,
   };
 
-  return flag;
+  if (!flag.isEnabled) {
+    console.warn("Flag is not enabled:", flag.name);
+    return false;
+  }
```

Comment:
Not sure if it belongs here or elsewhere, but should this check if the flag hasn't been soft deleted?
Alternately, we could auto-disable experiments when soft deleting them in the database and then the `.isEnabled` check would ignore deleted experiments.

Reply:
We haven't fully implemented this yet (e.g. there's no way to do this in the UI) but I'll add the check for it here anyway, thanks!